docs/notes/versions.rst

```diff
@@ -427,9 +427,9 @@ Fast DDS as the core middleware.
             * - Product
               - Related version
             * - `Fast DDS Gen <https://github.com/eProsima/Fast-DDS-Gen/>`__
-              - `v4.0.2 <https://github.com/eProsima/Fast-DDS-Gen/releases/tag/v4.0.2>`__
+              - `v4.0.3 <https://github.com/eProsima/Fast-DDS-Gen/releases/tag/v4.0.3>`__
             * - `Fast DDS Gen - IDL parser <https://github.com/eProsima/IDL-Parser/>`__
-              - `v4.0.2 <https://github.com/eProsima/IDL-Parser/releases/tag/v4.0.2>`__
+              - `v4.0.3 <https://github.com/eProsima/IDL-Parser/releases/tag/v4.0.3>`__
             * - `Fast DDS python <https://github.com/eProsima/Fast-DDS-python/>`__
               - `v2.1.0 <https://github.com/eProsima/Fast-DDS-python/releases/tag/v2.1.0>`__
             * - `Shapes Demo <https://github.com/eProsima/ShapesDemo/>`__
```

Comment:
Should we also bump Shapes Demo in this PR to `v3.1.1` ?

Reply:
Definetely!

Reply:
7e9b9f5